PySDM/physics/isotope_kinetic_fractionation_factors/jouzel_and_merlivat_1984.py

```diff
@@ -0,0 +1,13 @@
+"""
+kinetic fractionation factor from [Jouzel & Merlivat 1984](https://doi.org/10.1029/JD089iD07p11749)
+"""
+
+
+class JouzelAndMerlivat1984:  # pylint: disable=too-few-public-methods
+    @staticmethod
+    def alpha_kinetic(alpha_equilibrium, relative_humidity, D_heavy2D_light):
```

Comment:
D might look like Deuterium - perhaps better to name it `heavy_to_light_diffusivity_ratio`?